include/TrustWalletCore/TWCoinType.h

```diff
@@ -104,6 +104,7 @@ enum TWCoinType {
     TWCoinTypeBoba = 10000288,
     TWCoinTypeMetis = 1001088,
     TWCoinTypeAurora = 1323161554,
+    TWCoinTypeSyscoinNEVM = 57,
```

Comment:
For EVM-clones which don't have proper coinID, we derive it from chainID, by adding 10000000, so it should be `10000057`.